docs/content/learn/tutorials/component-definition/_index.md

```diff
@@ -0,0 +1,680 @@
+- `components` (optional) - Groups `component` items which define given component(s) in the component definition. Zero or more `component` items may be used.
+- `capabilities` (optional) - Defines a group of given capabilities in the component definition. Zero or more `capability` items may be used. Capabilities are not covered in this tutorial.
+- `back-matter` (optional) – Contains references used within the component definition. Use of `back-matter` is not covered in this tutorial.
+{{% /tab %}}
+{{< /tabs >}}
+
+For simplicity of this tutorial, we will only discuss certain data structures in the sections that follow and identify how they can be used to represent our component definition.
+
+### Defining the Component Definition’s Metadata
+
+Most OSCAL models have a standard metadata syntax, therefore, this is not covered extensively in this tutorial.  There are a few considerations however when authoring metadata for a component definition, such as the `<role>`, `<location>`, and `<party>` elements illustrated in lines 9-18 below.
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+{{< highlight xml "linenos=table" >}}
+<component-definition xmlns="http://csrc.nist.gov/ns/oscal/1.0"
+    uuid="a7ba800c-a432-44cd-9075-0862cd66da6b">
+  ...  
+	<metadata>
+		<title>MongoDB Component Definition Example</title>
+		<last-modified>2001-12-17T09:30:47Z</last-modified>
+		<version>20210507</version>
+		<oscal-version>1.0.0-rc2</oscal-version>
+		<role id="supplier">
+			<title>Supplier</title>
+		</role>
+		<location uuid="0db91129-35ed-474f-a25e-c3ff1e2cb9dc">
+			<address />
+			<url>https://www.mongodb.com</url>
+		</location>
+		<party uuid="ef7c799a-c50e-49ab-83e0-515e989e6df1" type="prepared-for">
+		    <name>MongoDB</name>
+		</party>
+	</metadata>
+  ...
+</component-definition>
+{{< /highlight >}}
+
+The [`<role>`](/reference/latest/component-definition/xml-reference/#/component-definition/metadata/role) element defines a function assumed or expected to be assumed by a party in a specific situation.  Common examples include [component definition authors](/concepts/layer/implementation/component-definition/#component-definition-authors) (e.g., Product and Service Vendors, Capability Owners, Policy and Process Owners, Standards Bodies and Validation Bodies) and [component definition consumers](/concepts/layer/implementation/component-definition/#component-definition-consumers) (e.g., System Owners and SSP Authors).  This element has a required `@id` attribute which expects an [NCName data type](/reference/datatypes/#ncname).  The `<location>` element can be used to associate the metadata with a location, including physical addresses and urls.  The `<location>` element has a `@uuid` attribute which requires a UUID.  Lastly, the `<party>` element represents either a person or organization that serves as the responsible entity.  There are prescribed types, including *"prepared-for"*, *"prepared-by"*, and *"approved-by"*.
```

Comment:
URLS are not intended to be locations. Similar issue for `prepared-for`. See above. We should discuss a better way to capture this.